installer/resources/lambda_rule_engine/files/rule_engine_cloudwatch_rules.json

```diff
@@ -6752,5 +6752,27 @@
     "modifiedDate": "2022-07-19",
     "severity": "medium",
     "category": "security"
+  },
+  {
+    "ruleId": "Disable_Contained_DB_Auth_Flag_for_SQL_Server_DB_Instances",
+    "ruleUUID": "Disable_Contained_DB_Auth_Flag_for_SQL_Server_DB_Instances",
+    "policyId": "Disable_Contained_DB_Auth_Flag_for_SQL_Server_DB_Instances",
+    "ruleName": "Disable Contained Database Authentication Flag for SQL Server Database Instances",
+    "targetType": "cloudsql_sqlserver",
+    "assetGroup": "gcp",
+    "alexaKeyword": "sqlserver",
+    "ruleParams": "{\"params\":[{\"encrypt\":false,\"value\":\"disable-contained-database-Auth-flag-for-sql-server\",\"key\":\"ruleKey\"},{\"key\":\"dataBaseType\",\"value\":\"SQLSERVER\",\"isValueNew\":true,\"encrypt\":false},{\"key\":\"dbFlagName\",\"value\":\"contained database authentication\",\"isValueNew\":true,\"encrypt\":false},{\"key\":\"policyName\",\"value\":\"Use CMK to encrypt GKE Cluster Node\",\"isValueNew\":true,\"encrypt\":false},{\"encrypt\":false,\"value\":\"medium\",\"key\":\"severity\"},{\"encrypt\":false,\"value\":\"security\",\"key\":\"ruleCategory\"},{\"encrypt\":false,\"value\":\"\",\"key\":\"ruleOwner\"}],\"environmentVariables\":[],\"ruleId\":\"Disable_Contained_DB_Auth_Flag_for_SQL_Server_DB_Instances\",\"autofix\":false,\"alexaKeyword\":\"sqlserver\",\"ruleRestUrl\":\"\",\"targetType\":\"cloudsql_sqlserver\",\"pac_ds\":\"gcp\",\"policyId\":\"Disable_Contained_DB_Auth_Flag_for_SQL_Server_DB_Instances\",\"assetGroup\":\"gcp\",\"ruleUUID\":\"Disable_Contained_DB_Auth_Flag_for_SQL_Server_DB_Instances\",\"ruleType\":\"ManageRule\"}",
```

Comment:
Can we also parameterise the dbFlagValue like the dbFalgName.

Reply:
sure dheeraj we can do that